client/app/components/dialog.vue

```diff
@@ -1,48 +1,70 @@
 <template lang="pug">
-  cozy-dialog(v-if="content",
```

Comment:
@m4dz : I used this synthax to override footer or header content; I seems to me that this feature is lost by this change. May I miss something?

Furthermore I don't see in your markup where your are using `content` property; how do you add specific `KonnectorComponent`  to global `DialogComponent`?